### DISCUSSION TITLE
xbuild standalone APKs and IPAs?

### DISCUSSION THREAD

**Discussion by maybeJosiah** [rust-mobile/xbuild#193] [Q&A]
Is it possible to build standalone .apk and .ipa files with xbuild and could you document how if there is? Also, can I use Wgpu, Winit, and Egui with this? I hear there is APK signing, does xbuild support that?

**Comment by dvc94ch** [comment]
You should be able to do all those things if you dig in. It's been too long since I've done anything with xbuild so don't recall the details
